### DISCUSSION TITLE
Allow users to click enter on the welcome naming screen

### DISCUSSION THREAD

**Discussion by hkamran80** [WebCrateApp/webcrate#43] [Ideas] [answered]
When you first start up WebCrate, it prompts you to name your instance. Currently, after you enter a name, you have to click the "Next Step" button. It makes more sense to allow the user to click enter to proceed to the next page, in my opinion. I've made this change in [my fork](https://github.com/hkamran80/webcrate/tree/develop). What do you think?

**Comment by BetaHuhn** [comment, marked as answer]
Yeah that makes sense. Pressing enter is already used in other parts of the app to submit actions, like when adding a new link.

Instead of `keyup.enter`, I'd prefer it if you use the `vue-shortkey` directive which is already used in other parts of the app as well:

https://github.com/WebCrateApp/webcrate/blob/6f3321d900d31a841187decf064fe7f172226e8d/client/components/Modal/AddLink.vue#L22

If you create a PR with that change I'll merge it and include it in the next release.

**Reply by hkamran80** [reply]
Made the PR (#45)!

**Reply by BetaHuhn** [reply]
This is included in version [1.4.1](https://github.com/WebCrateApp/webcrate/releases/tag/v1.4.1) :tada:
